assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/TotalUserCount.js

```diff
@@ -39,11 +39,20 @@ import { getAvailableDateRanges } from '../../../../../util/date-range';
 import ChangeArrow from '../../../../../components/ChangeArrow';
 import PreviewBlock from '../../../../../components/PreviewBlock';
 import ReportError from '../../../../../components/ReportError';
-const { useSelect } = Data;
+import { CORE_FORMS } from '../../../../../googlesitekit/datastore/forms/constants';
+import { FORM_ALL_TRAFFIC_WIDGET } from '../../../datastore/constants';
+const { useSelect, useDispatch } = Data;
 
 export default function TotalUserCount( { loaded, error, report, dimensionValue } ) {
 	const dateRange = useSelect( ( select ) => select( CORE_USER ).getDateRange() );
 
+	const { setValues } = useDispatch( CORE_FORMS );
+	const showAllUsers = ( e ) => {
+		e.preventDefault();
```

Comment:
I don't think we need to call `preventDefault`, we don't typically do this in other button listeners either. It's only necessary to actually prevent default behavior that would otherwise run, e.g. a form submission or opening a link.